scripts/stop-dev.sh

```
@@ -0,0 +1,96 @@
+        fi
+    else
+        print_status "No processes running on port $port"
+    fi
+}
+
+# Function to find and kill all node processes related to this project
+kill_project_processes() {
+    print_status "Searching for related Node.js processes..."
+    
+    # Find processes containing todo-app, next, or express keywords
+    local project_processes=$(ps aux | grep -E "(todo-app|next|express|pnpm.*dev)" | grep -v grep | awk '{print $2}' || true)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)